packages/amazon-sumerian-hosts-babylon/package.json

```diff
@@ -17,7 +17,7 @@
     "test": "node ../../node_modules/karma/bin/karma start",
     "build": "webpack --config webpack.common.js",
     "build-test": "npm run build; npm run test",
-    "start": "webpack-dev-server --open --config webpack.dev.js",
+    "examples-serve": "webpack-dev-server --config webpack.dev.js",
     "build-start": "npm run build; npm run start",
```

Comment:
build-start also needs to change if we remove the start script

Reply:
Good catch. I will fix.

Reply:
This issue is addressed in the update I just posted. Running `npm start` will now open a browser displaying the package root contents.